### PR TITLE
Solving issues #5 and #6

### PR DESCRIPTION
**Pull request by Franco0700**
Also I added a message where it says the amount of ignored vulnerabilities.

### CODE REVIEW

**Comment by Franco0700** [comment]
@opeco17

File: poetry_audit_plugin/command.py [hunk 1]
```diff
@@ -131,6 +141,27 @@ def get_json_report(self, vulnerabilities: List[Vulnerability]) -> str:
         }
         return json.dumps(json_report_dict, indent=2)
 
+    def iter_vulner(self, vulnerabilities):
+        filtered = []
+        ignored_vulns = 0
+        for vulner in vulnerabilities:
+            if self.option("ignore-package"):
+                ignored_packages = self.option("ignore-package").split(',')
```
Comment (dgutson) [inline poetry_audit_plugin/command.py:149]: pls avoid re-evaluating this on each iteration.

File: poetry_audit_plugin/command.py [hunk 2]
```diff
@@ -131,6 +141,39 @@ def get_json_report(self, vulnerabilities: List[Vulnerability]) -> str:
         }
         return json.dumps(json_report_dict, indent=2)
 
+    def get_suppressions(self):
+        ignored_packages, codes = None, None
```
Comment (dgutson) [inline poetry_audit_plugin/command.py:145]: do they charge per keystroke? :)

File: poetry_audit_plugin/command.py [hunk 3]
```diff
@@ -131,6 +141,39 @@ def get_json_report(self, vulnerabilities: List[Vulnerability]) -> str:
...
+            else:
+                ignored_vulns += 1
+        return ignored_vulns, new_details
+
+    def iter_vulner(self, vulnerabilities):
+        filtered = []
+        ignored_vulns = 0
+        ignored_packages, codes = self.get_suppressions()
+        for vulner in vulnerabilities:
+            if self.option("ignore-package"):
```
Comment (dgutson) [inline poetry_audit_plugin/command.py:167]: move this to a boolean flag

File: poetry_audit_plugin/command.py [hunk 4]
```diff
@@ -131,6 +141,39 @@ def get_json_report(self, vulnerabilities: List[Vulnerability]) -> str:
...
+                if vulner.name in ignored_packages:
+                    ignored_vulns += 1
+                    continue
+            if self.option("ignore-code"):
```
Comment (dgutson) [inline poetry_audit_plugin/command.py:171]: ditto, parse options at the beginning

File: tests/test_main.py [hunk 5]
```diff
@@ -119,3 +119,95 @@ def test_vulnerabilities_in_main_dev_json_report(tmp_path: Path) -> None:
...
+    result = run_audit(testing_dir=testing_dir, args=["--json", "--ignore-package=ansible-tower-cli"])
+    result_dict = json.loads(result.stdout)
+    vulnerabilitie_names = []
+    vulnerabilitie_cve = []
+    for vuln in result_dict["vulnerabilities"]:
+        vulnerabilitie_names.append(vuln["name"])
+
+    assert dev_vulnerability in vulnerabilitie_names
+    assert "ansible-tower-cli" not in vulnerabilitie_names
+    assert result.returncode == 1
```
Comment (dgutson) [inline tests/test_main.py:213]: pls add the EOL

**Comment by opeco17** [comment]
@Franco0700 
Thank you for implementing the feature
Let me prepare to release new version after fixing styles

**Comment by opeco17** [comment]
@dgutson 
Thank you for reviewing PR
It's very helpful

**Pull request merged**